web/src/narrow_state.ts

```diff
@@ -25,6 +25,14 @@ export function search_terms(current_filter: Filter | undefined = filter()): Nar
     return current_filter.terms();
 }
 
+export function is_search_view(): boolean {
+    const current_filter = filter();
+    if (current_filter && !current_filter.supports_collapsing_recipients()) {
+        return true;
+    }
+    return false;
+}
+
```

Comment:
Looking at the other methods described in this module, a common pattern here is to accept the filter as an argument with a fallback when that parameter is not passed: `current_filter: Filter | undefined = filter()`. I really like this logic, which allows the method to be more flexible, and I don't see a reason why we wouldn't want the same behavior in this method as well.

Reply:
I agree with making the function aligns with rest of the functions in the module.
but what benefit do we gain by accepting the filter as an argument instead, flexible in what way ? can you give an example please

Reply:
You should be able to `git grep` for these functions being called with a parameter. 

Also, just noting the communication here was poor; you asked the reviewer to do extra work that should have been easy for you to do yourself, and also didn't specify that you in fact made the change when pushing the PR. 

You can greatly increase the speed with which your work is reviewed and merged if you work on communication skills as well as on spending more time trying to answer questions like this yourself rather than asking them of reviewers.